Example/Tests/GINISupportedFormatsViewControllerTests.swift

```diff
@@ -68,6 +68,31 @@ final class GINISupportedFormatsViewControllerTests: XCTestCase {
         XCTAssertEqual(sectionItemsCount, tableViewSectionItemsCount, "section items count and table section items count should be always equal")
     }
     
+    func testFirstSectionItemsCountFileImportDisabled() {
+        GiniConfiguration.sharedConfiguration.fileImportSupportedTypes = .none
+
+        supportedFormatsViewController = SupportedFormatsViewController()
+        _ = supportedFormatsViewController.view
+        
+        let section1items = supportedFormatsViewController.tableView(supportedFormatsViewController.tableView, numberOfRowsInSection: 0)
+        
+        XCTAssertEqual(section1items, 1, "items count in section 1 should be 1 when file import is disabled")
+        GiniConfiguration.sharedConfiguration.fileImportSupportedTypes = .pdf_and_images
```

Comment:
Why here?
If you are afraid of leaving an incorrect config entry at the end of a test, use dependency injection. We talked about it several days ago.